contracts/src/v0.8/automation/chains/OptimismModule.sol

```diff
@@ -4,6 +4,10 @@ pragma solidity 0.8.19;
 import {OVM_GasPriceOracle} from "../../vendor/@eth-optimism/contracts/v0.8.9/contracts/L2/predeploys/OVM_GasPriceOracle.sol";
 import {ChainModuleBase} from "./ChainModuleBase.sol";
 
+/**
+ * @notice This contract is deprecated. Please use OptimismModuleV2 which utilizes the most recent offerings from OP
```

Comment:
do we need to keep this contract around? Should we just make the "V2" contract the one and only `OptimismModule`?

Reply:
will remove after V2 is launched